CHANGES.md

```diff
@@ -8,6 +8,9 @@
 
  * Bump minimum version requirement of `base` to 4.5
 
+ * Add `genericHashWithSalt` and `genericLiftHashWithSalt`;
+   export Generics internals from `Data.Hashable.Generic` module.
```

Comment:
s/from/from the/

Reply:
while at it, please move it one bullet-point up, so it's alongside the other API addition